src/components/simulation/SimulationButton.vue

```diff
@@ -2,22 +2,25 @@
   <div class="simulationButton">
     <v-menu :close-on-content-click="false" offset-y>
       <template #activator="{ on }">
-        <div class="split-btn">
+        <div class="btn-split text-no-wrap">
           <v-btn
             :disabled="state.disabled"
             :loading="state.project.simulation.state.running"
             @click="state.project.startSimulation()"
-            class="main-btn"
+            class="btn-main"
             outlined
+            title="Simulate"
           >
             <v-icon left>mdi-play</v-icon>
```

Comment:
`left` prevents it from getting centered:
![grafik](https://user-images.githubusercontent.com/53972736/205667022-f3b30a78-f70b-42fe-928a-bc41ff52653e.png)
Therefore, I would suggest
```suggestion
            <v-icon center>mdi-play</v-icon>
```
![grafik](https://user-images.githubusercontent.com/53972736/205667261-eb6c2a68-e5f5-4644-b84b-b467edb58cff.png)
This would also increase the size of the play button. I would prefer that, since it should be a bit more significant than the arrow down.